tests/journey/test_federal_senate_dataset.py

```diff
@@ -0,0 +1,37 @@
+import os
+from tempfile import gettempdir
+from unittest import main, skipIf, TestCase
+
+
+from serenata_toolbox.federal_senate.federal_senate_dataset import FederalSenateDataset
+
+
+class TestJourneyFederalSenateDataset(TestCase):
+    def setUp(self):
+        self.path = gettempdir()
+        self.subject = FederalSenateDataset(self.path)
+
+    @skipIf(os.environ.get('RUN_INTEGRATION_TESTS') != '1',
+            'Skipping integration test')
+    def test_journey_federal_senate_dataset(self):
+        # fetch_saves_raw_files
+        self.subject.fetch()
+        federal_senate_csv_files = ['federal-senate-{}.csv'.format(year) for year in range(self.subject.FIRST_YEAR, self.subject.NEXT_YEAR)]
```

Comment:
I would create an variable that has the year range just so you won't have to repeat `range(self.subject.FIRST_YEAR, self.subject.NEXT_YEAR)` and on line 26